docs/user/masternodes/setup-testnet.rst

```diff
@@ -774,17 +787,6 @@ response::
     "IsFailed": false
   }
 
-Tor
```

Comment:
> Real pity for a privacy centric coin to abandon providing full node services over TOR, please re-consider this.

Good point, I got stuck for a while due to incorrect dash.conf and assumed the Tor install wasn't working. This is readded before the Dash Core section now (which requires it to sync), but not tested yet.